content/docs/howtos/customize-i3-configuration.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Customize i3 Configuration"
+linkTitle: "How to Customize i3 Configuration"
+weight: 1
+description: >
+  How to make changes to the way i3 looks and behaves.
+---
+
+# How to work with Regolith 2's defaults
+
+The default configurations are stored in `/usr/share/regolith/i3/config.d`. These are loaded alphabetically. Then user configurations in `~/.config/regolith2/config.d` are loaded, also in alphabetical order. Regolith's default configuration is built to be customized by setting `Xresources` variables, adding user configuration, and adding or removing default configurations via `apt`. For this reason, there are several approaches to customization of i3 in Regolith that can be used separately or in combination to achieve your configuration cleanly:
+
+* Use `Xresources` to override variables, e.g. keybindings, strings, program names, colors, etc.
+* Add or remove Regolith default configuration files with `apt`
+* Add your own user configuration files in 
+
+## How to set an `Xresources` variable
+
+The following configuration example (from `/usr/share/regolith/i3/config.d/80_compositor`) can be customized to load whatever compositor you want, without editing or overriding the existing i3 configuration file, by setting `$i3-wm.program.compositor`.
+
+```
+set_from_resource $i3-wm.program.compositor i3-wm.program.compositor /usr/share/regolith-compositor/init
+exec_always --no-startup-id $i3-wm.program.compositor
+```
+
+To do this, simply add a line to `~/.config/regolith2/Xresources` like:
+
+```
+# Use my own home-made compositor
+i3-wm.program.compositor: <my-custom-compositor>
+```
+
+Similarly, you can override other i3 options with `Xresources` based on the variable names found in `/usr/share/regolith/i3/config.d` files:
+
+```
+## Gap configuration
+i3-wm.gaps.inner.size: 1
+
+## Border configuration
+i3-wm.window.border.size: 3
+i3-wm.client.focused.color.child_border: #AAD3E9
+
+## Workspace names
+i3-wm.workspace.01.name: 1:FOO
+```
+
+For more, please [read a more in-depth discussion on `Xresources`]({{< ref "docs/howtos/override-xres.md" >}})
+
+## How to uninstall a default partial
+
+If you want to wholly replace or remove settings provided by a default partial, you can use `apt` to remove the package which provides that partial.
+
+To identify the name of the package that provides the partial you want to remove:
+
+```console
+dpkg -S /usr/share/doc/regolith-i3-workspace-config
+```
+
+The output contains the name of the package:
+
+```
+regolith-i3-workspace-config: /usr/share/doc/regolith-i3-workspace-config
+```
```

Comment:
I admit I didn't look very hard for other examples, but what's the convention in this documentation for documenting a command and its output?

Reply:
I.e. prefix the commands with a `$` and include the output in same code block like:

```
$ command
the output of the command
...
more important output
```

?

Reply:
This has been a point of contention in the past.  People seem to prefer w/out the `$` prefix.  Good as is.